.ado/publish.yml

```diff
@@ -6,14 +6,14 @@ variables:
   - template: variables/vs2019.yml
 
 schedules:
-- cron: "0 0 * * *"
-  displayName: Nightly publish build
+- cron: "0 5 * * *" # 5AM Daily UTC (10PM Daily PST)
+  displayName: Nightly master publish build
   branches:
     include:
-    - master
+      - master
 
-trigger:
-  batch: true
+- cron: "0 5 * * 1" # 5AM Monday UTC (10PM Sunday PST)
```

Comment:
Late Sunday is good for capturing all the work in the week, but also means we send release notifications at 10pm Sunday where folks are less likely to see them. Open to thoughts on the best time here.